assets/css/_sass/website/_shame.scss

```diff
@@ -5,4 +5,14 @@
   thead th {
     padding-top: 0;
   }
+}
+
+/* 
+* This will be removed when we move to version X.X (laura help me here) of the helix-ui code. The version currently used on the site does not have styling for the hxHeading-5 class. 
+*/
+
+.hxHeading5 {
+	font-size: 0.75rem;
```

Comment:
Spacing is a bit weird here. Check indent.

Reply:
updated